modules/kubernetes-addons/argo-rollouts/locals.tf

```diff
@@ -0,0 +1,23 @@
+locals {
+
+  name                 = "argo-rollouts"
+  service_account_name = "${local.name}-sa"
+  default_helm_config = {
+    name        = local.name
+    chart       = local.name
+    repository  = "https://argoproj.github.io/argo-helm"
+    version     = "2.9.1"
+    namespace   = local.name
+    description = "Argo Rollouts AddOn Helm Chart"
```

Comment:
Add these for defaults or we can add these three defaults to `helm_addon` using `try` function

```hcl
    timeout                    = "300"
    create_namespace           = false
    values                     = local.default_helm_values
```

Reply:
yes, good catch.